docs/reference/Custom Resources/verifiers.md

```diff
@@ -0,0 +1,22 @@
+# Verifier
+
```

Comment:
question about the path.  Should the path at lower case to be consistent with external plugins dir. If yes, we do need category.json?

Reply:
I've updated the file paths to now be consistently lower case. For upper-case sub folders, I've added corresponding `_category_.json`